.github/workflows/kind.yml

```diff
@@ -395,6 +395,32 @@ jobs:
         path: antrea-ubuntu.tar
         retention-days: 1 # minimum value, in case artifact deletion by 'artifact-cleanup' job fails
 
+  test-netpol-cyclonus:
```

Comment:
@mattfenwick as mentioned on Slack, I don't think we want to run this job on every PR, but rather as a daily Cron job. We can reconsider after we have more data about stability of the job in Kind and how long the job takes care. You will need to place this job in its own Github workflow file, since triggers are defined at the file-level.

Reply:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onschedule

Reply:
Working on this ... +1

Reply:
👍